lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -745,9 +748,15 @@ public static List<ICommunicationGroupClient> CommGroupClients(string groupName,
         public static NetworkService<GroupCommunicationMessage> BuildNetworkService(
             IPEndPoint nameServerEndpoint, IObserver<NsMessage<GroupCommunicationMessage>> handler)
         {
-            var remoteManagerFactory = TangFactory.GetTang().NewInjector().GetInstance<IRemoteManagerFactory>();
+            var injector = TangFactory.GetTang().NewInjector();
+            var remoteManagerFactory = injector.GetInstance<IRemoteManagerFactory>();
             return new NetworkService<GroupCommunicationMessage>(
```

Comment:
Actually, `NetworkService` is injectable. In which case we should use Tang to create the instance here.